src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -2955,99 +2955,107 @@ class Project: ADOSVTBase
+                        $controlResult.AdditionalInfoInCSV = $groups -join ' ; '
                     }
-                    if ($foundRestrictedPermissions) {
-                        $restrictedGroups += $group
+                    else {
+                        $controlResult.AddMessage([VerificationResult]::Passed, "Build pipelines are not allowed to inherit excessive permissions for a broad group of users at project level.");
                     }
-                    $foundRestrictedPermissions = $false
-                }
-
-                if($restrictedGroups.Count -ne 0)
-                {
-                    $accessList = $restrictedGroups | Select-Object -Property @{Name="Name"; Expression = {$_.IdentityName}}, @{Name="Type"; Expression = {$_.IdentityType}}, @{Name="Permissions"; Expression = {$_.Permissions.Name}}
-                    $controlResult.AddMessage([VerificationResult]::Failed,"Count of broader groups that have excessive permissions to repository at a project level: $($restrictedGroups.Count)");
-                    $controlResult.AddMessage("Validate that the following broader groups that have excessive permissions to repositories: `n", $($accessList | FT -AutoSize | Out-String -Width 512));
-                    $stateData = ($responseObj.dataProviders."ms.vss-admin-web.security-view-members-data-provider".identities | Select-Object -Property @{Name="Name"; Expression = {$_.displayName}},@{Name="Type"; Expression = {$_.subjectKind}},@{Name="Scope"; Expression = {$_.Scope}}) 
-                    $controlResult.SetStateData("List of broader groups having access to repositories: ", $stateData);
-                    $controlResult.AdditionalInfo += "Count of broader groups that have excessive permissions to repository at a project level: $($restrictedGroups.Count)";
-
                 }
                 else
                 {
                     $controlResult.AddMessage([VerificationResult]::Passed,"No broader groups have excessive access to repository at a project level.");
                 }
-                $controlResult.AddMessage("`nNote:`nThe following groups are considered 'broad' and should not have excessive privileges: `n$($restrictedBroaderGroupsForRepo | FT | out-string)");
-                $responseObj = $null;
             }
-            else 
+            else
             {
-                $controlResult.AddMessage([VerificationResult]::Error, "List of restricted broader groups and restricted roles for repository is not defined in the control settings for your organization policy.");
+                $controlResult.AddMessage([VerificationResult]::Error,"Unable to fetch repositories permission details at a project level.");
             }
+            $controlResult.AddMessage("`nNote:`nThe following groups are considered 'broad' and should not have excessive privileges: `n$($restrictedBroaderGroupsForRepo | FT | out-string)");
+            $responseObj = $null;
```

Comment:
should we also add list of excessive permissions : the following permissions are considered excessive